addon/ng2/commands/test.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var chalk        = require('chalk');
+var Command      = require('ember-cli/lib/models/command');
+var Promise      = require('ember-cli/lib/ext/promise');
+var Project      = require('ember-cli/lib/models/project');
+var SilentError        = require('silent-error');
+var validProjectName   = require('ember-cli/lib/utilities/valid-project-name');
+var normalizeBlueprint = require('ember-cli/lib/utilities/normalize-blueprint-option');
+
+var TestCommand = require('ember-cli/lib/commands/test');
+var childProcess = require('child_process')
+
+module.exports = TestCommand.extend({
+
+    init: function(){
+      var cwd = process.cwd();
+      this.karma = this.karma || require(cwd + '/node_modules/karma/lib/index');
```

Comment:
@IgorMinar I'm not familiar w/ `require.resolve` are you suggesting that I use do this:

`this.karma = this.karma || require(require.resolve(cwd + '/node_modules/karma/lib/index'));`